specification/elasticsan/resource-manager/readme.md

```diff
@@ -0,0 +1,74 @@
+# storagepool
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for storagepool.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the storagepool.
+
+```yaml
+openapi-type: arm
+tag: package-2021-11-20-preview
+```
+
+### Tag: package-2021-08-22-preview
```

Comment:
It seems the tag version is not correct.
should be "package-2021-11-20-preview"

Reply:
Thank you @blueww ! I just have raised a new PR https://github.com/Azure/azure-rest-api-specs/pull/18731 to address this issue. Resolving this comment as this PR is already in closed state.